erpnext/hr/doctype/employee_onboarding/employee_onboarding.js

```diff
@@ -40,21 +38,6 @@ frappe.ui.form.on('Employee Onboarding', {
 			}, __('Create'));
 			frm.page.set_inner_btn_group_as_primary(__('Create'));
 		}
-		if (frm.doc.docstatus === 1 && frm.doc.project) {
-			frappe.call({
-				method: "erpnext.hr.utils.get_boarding_status",
-				args: {
-					"project": frm.doc.project
-				},
-				callback: function(r) {
-					if (r.message) {
-						frm.set_value('boarding_status', r.message);
-					}
-					refresh_field("boarding_status");
-				}
-			});
-		}
-
```

Comment:
why remove this?

Reply:
Suppose Hr removed changed the status of Employee-onboarding but on refresh, it was fetching the boarding status as per project and setting the value leaving the form to Unsaved state. which is not good UI.
I checked there is already a server-side validation for completion. making this command useless.